packages/fabric-website/gulpfile.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+let build = require('@microsoft/web-library-build');
+let gulp = require('gulp');
+let buildConfig = build.getConfig();
+let distFolder = buildConfig.distFolder;
+let path = require('path');
+let packageFolder = buildConfig.packageFolder || '';
+let isProduction = process.argv.indexOf('--production') >= 0;
+let isNuke = process.argv.indexOf('nuke') >= 0;
+
+
+/** @todo: disable lint config. */
+build.tslint.setConfig({
+  lintConfig: require('./tslint.json'),
+  sourceMatch: ['src/**/*.ts', 'src/**/*.tsx', '!src/**/*.scss.tsx']
+});
+
+/* Configure TypeScript 2.0. */
+build.typescript.setConfig({ typescript: require('typescript') });
```

Comment:
Get rid of this if you're just using the version of web-library-build that comes with Typescript 2.1